puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/testKatello.groovy

```diff
@@ -118,6 +118,11 @@ pipeline {
 
             }
         }
+        stage('Cleanup RVM') {
+            steps {
+                cleanupRVM(ruby)
```

Comment:
The post section below already calls `cleanup` which in turn calls cleanupRVM. So if gemsets are not being cleaned up I'd think its due to a different problem. Can you point to a failure to do this?